com.unity.ml-agents/Runtime/ActionMasker.cs

```diff
@@ -19,6 +19,43 @@ internal ActionMasker(BrainParameters brainParameters)
             m_BrainParameters = brainParameters;
         }
 
+        /// <summary>
+        /// Sets an action mask for discrete control agents. When used, the agent will not be
+        /// able to perform the action passed as argument at the next decision. If no branch is
```

Comment:
In this doc, can you just say that for this method the branch is assumed to be 0 instead of saying "if no branch is specified."